aodntools/ncwriter/template.py

```diff
@@ -295,6 +295,11 @@ def create_variables(self, **kwargs):
         **kwargs are included here to overload all options for all variables
         like `zlib` and friends.
         """
+
+        # variable attributes to convert to the same type as the variable
+        # datatype
+        varattrs_to_convert_to_datatype = ['_FillValue', 'valid_min', 'valid_max', 'valid_range']
```

Comment:
Don't need to include '_FillValue' as it is handled by the NetCDF4 library when it creates the variable.